_content/monographs/lamp-for-path-to-enlightenment_dipamkara-atisha.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "The Lamp for the Path to Enlightenment"
+authors:
+  - "Atīśa Dīpaṃkara Śrījñāna"
+editor: "Jamgön Kongtrul Lodrö Thaye"
+translator: "Khenpo David Karma Choephel"
+external_url: "https://dharmaebooks.org/the-lamp-for-the-path-to-enlightenment/"
```

Comment:
Where are the drive_links?

https://drive.google.com/file/d/1gIOnGs_LTTSmwwUtKVfblJrq5mFdZm7g/view?usp=drivesdk

https://drive.google.com/file/d/1-SpKvPvTigYQwROakv4mzLd92ubS1NuA/view?usp=drivesdk